Library/Formula/openssl.rb

```diff
@@ -120,10 +133,8 @@ def post_install
     openssldir.mkpath
     (openssldir/"cert.pem").atomic_write `security find-certificate -a -p #{keychains.join(" ")}`
 
-    if MacOS.version == :yosemite
-      (openssldir/"certs").install resource("Equifax_CA")
-      system bin/"c_rehash"
-    end
+    # Remove this once 1.0.2b lands.
+    rm_f openssldir/"certs/Equifax_CA" if MacOS.version == :yosemite
```

Comment:
It generates a symlink rather than a copy. If you kill the actual cert, the symlink becomes ineffective. If the symlink is named ` 578d5c04.0 ` on your machine as well, I'll add removing it to the formula.